commercial/app/controllers/commercial/ContentApiOffersController.scala

```diff
@@ -17,6 +17,12 @@ object ContentApiOffersController extends Controller with ExecutionContexts with
     "foundation-supported" -> "fc-container--foundation-supported"
   )
 
+  private val sponsorTypeToClassRefactor = Map(
```

Comment:
We could use a `case object` here as with our Switches i.e. `On` and `Off`. It would abstract away from string matching, which I think would be a good thing...

e.g.
```
sealed trait SponsorType
case object PaidFor extends SponsorType { override def toString = "paidfor"}
case object Sponsored extends SponsorType { override def toString = "supported"}
case object Foundation extends SponsorType { override def toString = "supported"}
```

we can then refer to `SponsorType.PaidFor` etc